team_b/yappy/lib/industry_menu.dart

```diff
@@ -1,17 +1,52 @@
 import 'package:flutter/material.dart';
 
-class MenuItem {
-  final String title;
-  final IconData icon;
-
-  MenuItem({required this.title, required this.icon});
-}
 class IndustryMenu extends StatelessWidget {
   final String title;
   final IconData icon;
 
   const IndustryMenu({required this.title, required this.icon, Key? key}) : super(key: key); 
 
+    Widget generateTranscript(BuildContext context, String title, String content) {
```

Comment:
Should just be ` const IndustryMenu({required this.title, required this.icon, super.key});`